File: specification/mobilenetwork/resource-manager/Microsoft.MobileNetwork/stable/2024-02-01/examples-java/DataNetworkDelete.java

/**
 * Samples for DataNetworks Delete.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/mobilenetwork/resource-manager/Microsoft.MobileNetwork/stable/2024-02-01/examples/DataNetworkDelete
     * .json
     */
    /**
     * Sample code: Delete data network.
     * 
     * @param manager Entry point to MobileNetworkManager.
     */
    public static void deleteDataNetwork(com.azure.resourcemanager.mobilenetwork.MobileNetworkManager manager) {
        manager.dataNetworks().delete("rg1", "testMobileNetwork", "testDataNetwork", com.azure.core.util.Context.NONE);
    }
}

File: specification/mobilenetwork/resource-manager/Microsoft.MobileNetwork/stable/2024-04-01/examples-js/DataNetworkDelete.js
const { MobileNetworkManagementClient } = require("@azure/arm-mobilenetwork");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Deletes the specified data network.
 *
 * @summary Deletes the specified data network.
 * x-ms-original-file: specification/mobilenetwork/resource-manager/Microsoft.MobileNetwork/stable/2024-04-01/examples/DataNetworkDelete.json
 */
async function deleteDataNetwork() {
  const subscriptionId =
    process.env["MOBILENETWORK_SUBSCRIPTION_ID"] || "00000000-0000-0000-0000-000000000000";
  const resourceGroupName = process.env["MOBILENETWORK_RESOURCE_GROUP"] || "rg1";
  const mobileNetworkName = "testMobileNetwork";
  const dataNetworkName = "testDataNetwork";
  const credential = new DefaultAzureCredential();
  const client = new MobileNetworkManagementClient(credential, subscriptionId);
  const result = await client.dataNetworks.beginDeleteAndWait(
    resourceGroupName,
    mobileNetworkName,
    dataNetworkName,
  );
  console.log(result);
}
